src/main/java/ui/Menu.java

```diff
@@ -6,7 +6,6 @@
 import java.awt.*;
 import java.awt.event.ActionEvent;
 import java.awt.event.ActionListener;
-import java.io.IOException;
 
 /**
  * Main menu screen once a user is logged in
```

Comment:
for the "Search Recipe" button, maybe change it to "View Recipe" to avoid confusion with the "search and sort recipe" button